File: Java/Events/PerformanceInfo.java
package LTD_API;

import java.util.Date;

public class PerformanceInfo
{
    private int performanceId;
    private Date performanceDate;
    private int ticketCount;
    private int totalAvailableTickesCount;
    private boolean containsDiscountOfferTickets;
    private boolean containsNoFeeOfferTickets;
    private decimal minimumTicketPrice;
    private int maximumConsecutiveSeatsCount;
    private int eventId;
    
    public int getPerformanceId()
    {
        return performanceId;
    }
    public void setPerformanceId(int performanceId)
    {
        this.performanceId = performanceId;
    }

    public Date getPerformanceDate()
    {
        return performanceDate;
    }
    public void setPerformanceDate(Date performanceDate)
    {
        this.performanceDate = performanceDate;
    }

    public int getTicketCount()
    {
        return ticketCount;
    }
    public void setTicketCount(int ticketCount)
    {
        this.ticketCount = ticketCount;
    }

    public int getTotalAvailableTickesCount()
    {
        return totalAvailableTickesCount;
    }
    public void setTotalAvailableTickesCount(int totalAvailableTickesCount)
    {
        this.totalAvailableTickesCount = totalAvailableTickesCount;
    }

    public boolean getContainsDiscountOfferTickets()
    {
        return containsDiscountOfferTickets;
    }
    public void setContainsDiscountOfferTickets(boolean containsDiscountOfferTickets)
    {
        this.containsDiscountOfferTickets = containsDiscountOfferTickets;
    }

    public boolean getContainsNoFeeOfferTickets()
    {
        return containsNoFeeOfferTickets;
    }
    public void setContainsNoFeeOfferTickets(boolean containsNoFeeOfferTickets)
    {
        this.containsNoFeeOfferTickets = containsNoFeeOfferTickets;
    }

    public decimal getMinimumTicketPrice()
    {
        return minimumTicketPrice;
    }
    public void setMinimumTicketPrice(decimal minimumTicketPrice)
    {
        this.minimumTicketPrice = minimumTicketPrice;
    }

    public int getMaximumConsecutiveSeatsCount()
    {
        return maximumConsecutiveSeatsCount;
    }
    public void setMaximumConsecutiveSeatsCount(int maximumConsecutiveSeatsCount)
    {
        this.maximumConsecutiveSeatsCount = maximumConsecutiveSeatsCount;
    }

    public int getEventId()
    {
        return eventId;
    }
    public void setEventId(int eventId)
    {
        this.eventId = eventId;
    }
}

File: PHP/Events/PerformanceInfo.php
<?php
namespace LTD_API;

class PerformanceInfo
{
    private $performanceId;
    //date format
    private $performanceDate;
    private $ticketCount;
    private $totalAvailableTickesCount;
    private $containsDiscountOfferTickets;
    private $containsNoFeeOfferTickets;
    private $minimumTicketPrice;
    private $maximumConsecutiveSeatsCount;
    private $eventId;
    
    public function getPerformanceId()
    {
        return $this->performanceId;
    }
    public function setPerformanceId($performanceId)
    {
        $this->performanceId = $performanceId;
    }

    public function getPerformanceDate()
    {
        return $this->performanceDate;
    }
    public function setPerformanceDate($performanceDate)
    {
        $this->performanceDate = $performanceDate;
    }

    public function getTicketCount()
    {
        return $this->ticketCount;
    }
    public function setTicketCount($ticketCount)
    {
        $this->ticketCount = $ticketCount;
    }

    public function getTotalAvailableTickesCount()
    {
        return $this->totalAvailableTickesCount;
    }
    public function setTotalAvailableTickesCount($totalAvailableTickesCount)
    {
        $this->totalAvailableTickesCount = $totalAvailableTickesCount;
    }

    public function getContainsDiscountOfferTickets()
    {
        return $this->containsDiscountOfferTickets;
    }
    public function setContainsDiscountOfferTickets($containsDiscountOfferTickets)
    {
        $this->containsDiscountOfferTickets = $containsDiscountOfferTickets;
    }

    public function getContainsNoFeeOfferTickets()
    {
        return $this->containsNoFeeOfferTickets;
    }
    public function setContainsNoFeeOfferTickets($containsNoFeeOfferTickets)
    {
        $this->containsNoFeeOfferTickets = $containsNoFeeOfferTickets;
    }

    public function getMinimumTicketPrice()
    {
        return $this->minimumTicketPrice;
    }
    public function setMinimumTicketPrice($minimumTicketPrice)
    {
        $this->minimumTicketPrice = $minimumTicketPrice;
    }

    public function getMaximumConsecutiveSeatsCount()
    {
        return $this->maximumConsecutiveSeatsCount;
    }
    public function setMaximumConsecutiveSeatsCount($maximumConsecutiveSeatsCount)
    {
        $this->maximumConsecutiveSeatsCount = $maximumConsecutiveSeatsCount;
    }

    public function getEventId()
    {
        return $this->eventId;
    }
    public function setEventId($eventId)
    {
        $this->eventId = $eventId;
    }
}

